sentry_sdk/integrations/aiohttp.py

```diff
@@ -48,6 +53,52 @@
 TRANSACTION_STYLE_VALUES = ("handler_name", "method_and_path_pattern")
 
 
+def create_trace_config():
+    # type: () -> TraceConfig
+    async def on_request_start(session, trace_config_ctx, params):
+        # type: (ClientSession, SimpleNamespace, TraceRequestStartParams) -> None
+        hub = Hub.current
+        if hub.get_integration(AioHttpIntegration) is None:
+            return
+
+        method = params.method.upper()
+        request_url = str(params.url)
```

Comment:
Let's run the url through `parse_url` like we do here https://github.com/getsentry/sentry-python/blob/d08ebdb9309e1b004c6f52202de58a09c2268e42/sentry_sdk/integrations/stdlib.py#L88-L96